spec.bs

```diff
@@ -219,6 +220,12 @@ problems.
         window such that the distance between the top and bottom edges of the
         viewport are |options|["{{DocumentPictureInPictureOptions/height}}"]
         pixels.
+<p class="note">
+The |options|["{{DocumentPictureInPictureOptions/ignorePreviousWindowBounds}}"]
+may be used as a hint to prefer behavior that is similar to steps 12 and 13,
```

Comment:
Can we be explicit and say something "If |options|["{{DocumentPictureInPictureOptions/ignorePreviousWindowBounds}}"] exists and is true, the user agent can use this hint to prefer...."

Reply:
done